examples/ray_multi_instance.py

```diff
@@ -17,18 +17,11 @@
     raise RayException.required_to("ray_multi_instance.py")
 
 
+from examples.argument_parser import default_argument_parser
```

Comment:
I take it the problem with `aiohttp` that meant we had to use relative imports no longer exists?  Is this the case?

Reply:
The latest version of `aiohttp` package does not cause issues with using the `examples` module. The `argument_parser` imports have thus been fixed.

Reply:
Good!